spec/lib/spree/active_shipping/bogus_carrier.rb

```diff
@@ -3,12 +3,15 @@ module ActiveShipping
     # Bogus carrier useful for testing.  For some reasons the plugin version of this class does not work
     # properly (it fails to return a RateResponse)
     class BogusCarrier < ::ActiveShipping::Carrier
+      include RSpec::Mocks::ExampleMethods
```

Comment:
I removed the duplicate in the gemspec, but I still need to include Rspec::Mocks because the find_rates function use instance_double to stub the RateResponse from ActiveShipping because it has a billion parameters otherwise

I switched the include for a require spec_helper instead, is it okay?